### ISSUE TITLE
Species Fungorum Plus - possible missing species

### ISSUE THREAD

**Issue by ManonGros** [CatalogueOfLife/data#518]
**Describe the problem**:

This is a message we originally received on the GBIF Helpdesk. It seems that those fungi species are missing from the GBIF backbone and the CoL:

Acarospora jenisejensis H. Magn., Svensk bot. Tidskr. 30: 248 (1936)
Caloplaca subpyracea (Nyl.) Zahlbr., Cat. Lich. Univers. 7: 185 (1931)
Cladonia terrae-novae f. cinerascens Ahti, Ann. bot. Soc. Zool.-Bot. fenn. Vanamo 32(no. 1): 82 (1961)
Fissurina leuconephela Nyl., Flora, Regensburg 52: 73 (1869)
Flavopunctelia darrowii (J.W. Thomson) Hale [as 'darrowi'], Mycotaxon 20(2): 682 (1984)
Graphis virginalis Tuck., in Eckfeldt, Bull. Torrey bot. Club 17: 256 (1890)
Heterodermia linearis Moberg & T.H. Nash [as 'lineare'], Bryologist 102(1): 7 (1999)
Paulia Fée, Linnaea 10: 471 (1836)
Pertusaria moreliensis B. de Lesd., Lich. Mexique: 18 (1914)
Pyrenula herrei Fink ex J. Hedrick, Mycologia 25(4): 309 (1933)
Usnea pallida Motyka, Usnea 2(1): 435 (1937)

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/F

**Literature references**:
(see above)

**Comment by yroskov** [comment]
Paul commented: "These are just ‘good names’ w.r.t. Index Fungorum".

**Comment by yroskov** [comment]
> It's still unclear to me, whether checklistbank software applied “bare name” status or Paul chosen it for taxonomically unvetted names

CLB software done this: names which have no relationship with taxa.

**Comment by mdoering** [comment]
Nearly 200.000 names in IF+ are bare names that don't get into COL !!!

<img width="250" alt="image" src="https://user-images.githubusercontent.com/327505/226556647-b6262a46-20cd-432a-a268-e91dbfc16660.png">

**Comment by mdoering** [comment]
The last ColDP file uploaded by @gdower had these verbatim records given:

> wc -l Name.tsv
>   508026 Name.tsv
> wc -l Taxon.tsv 
>   154571 Taxon.tsv
> wc -l Synonym.tsv 
>   158405 Synonym.tsv

That is 195.050 more names than taxa and synonyms together!

**Comment by mdoering** [comment]
A bare name is a lone name record without being referred to by a synonym or taxon.
It is useful for listing names that are not placed taxonomically anywhere.

But this simply looks like an error in generating the archive

**Comment by mdoering** [comment]
How is the coldp archive generated?

**Comment by yroskov** [comment]
> That is 195.050 more names than taxa and synonyms together!

Correct. Because Species Fungorum Plus includes both taxonomic checklist and also a set of taxonomically unvetted names from Index Fungorum. 

> How is the coldp archive generated?

Species Fungorum Plus CoLDP generated by Paul.

**Comment by mdoering** [comment]
> > It's still unclear to me, whether checklistbank software applied “bare name” status or Paul chosen it for taxonomically unvetted names
> 
> CLB software done this: names which have no relationship with taxa.

Well, CLB did what was given. There is nothing wrongly interpreted. It was given pure, "bare" names and they are stored as such. I do wonder if that amount of names are really unvetted and not to be included in COL

**Comment by yroskov** [comment]
Paul supplied names which have no relationship with taxa in CoLDP export. CLB software flagged them as "bare names" (there is no such status in Paul's ColDP).

**Comment by ManonGros** [comment]
The user who reported the original issue wrote to us to ask if the names could be integrated in the Catalogue of Life. I am not sure I fully understand the issue but is there a way to attach these names to taxa?

**Comment by mdoering** [comment]
We cannot integrate bare names that have no placement at all and no status assigned.
But there has been work started by @thomasstjerne to extract IF data through its API and that had more placed names if I understand. My hope would be to switch to this source as soon as possible to a) cover more names and b) have regular, monthly updates.

@thomasstjerne any idea what is needed to finish the work? A call with Paul?

**Comment by yroskov** [comment]
>is there a way to attach these names to taxa?

Taxonomically unresolved names ("bare names" according Markus) from Index Fungorum cannot be placed in taxa without expert knowledge/research. Mycologists should join their efforts in building global taxonomic checklists.

Just in case, I did not hear yet from Paul about his plans to update Species Fungorum in CoL via Index Fungorum API. I'll be happy to do it as soon as Paul approve this move. 

I had a look through Index Fungorum of 2022-10-15 (id 129659) on DEV. Looks like, it has no classification:   
https://www.dev.checklistbank.org/dataset/129659/classification. @thomasstjerne, is it a latest iteration of API crawling?

**Comment by thomasstjerne** [comment]
Seems the API crawler catches these names (links below). I think we should have a call with Paul soon.

[Acarospora jenisejensis H. Magn., Svensk bot. Tidskr. 30: 248 (1936)](https://www.dev.checklistbank.org/dataset/53548/taxon/410166)
[Caloplaca subpyracea (Nyl.) Zahlbr., Cat. Lich. Univers. 7: 185 (1931)](https://www.dev.checklistbank.org/dataset/53548/taxon/381805)
[Cladonia terrae-novae f. cinerascens Ahti, Ann. bot. Soc. Zool.-Bot. fenn. Vanamo 32(no. 1): 82 (1961)](https://www.dev.checklistbank.org/dataset/53548/taxon/365726) (Api crawl has the species, but not the form)
[Fissurina leuconephela Nyl., Flora, Regensburg 52: 73 (1869)](https://www.dev.checklistbank.org/dataset/53548/taxon/385021)
[Flavopunctelia darrowii (J.W. Thomson) Hale [as 'darrowi'], Mycotaxon 20(2): 682 (1984)](https://www.dev.checklistbank.org/dataset/53548/taxon/106818)
[Graphis virginalis Tuck., in Eckfeldt, Bull. Torrey bot. Club 17: 256 (1890)](https://www.dev.checklistbank.org/dataset/53548/taxon/386325)
[Heterodermia linearis Moberg & T.H. Nash [as 'lineare'], Bryologist 102(1): 7 (1999)](https://www.dev.checklistbank.org/dataset/53548/taxon/450409)
[Paulia Fée, Linnaea 10: 471 (1836)](https://www.dev.checklistbank.org/dataset/53548/taxon/3775)
[Pertusaria moreliensis B. de Lesd., Lich. Mexique: 18 (1914)](https://www.dev.checklistbank.org/dataset/53548/taxon/399717)
[Pyrenula herrei Fink ex J. Hedrick, Mycologia 25(4): 309 (1933)](https://www.dev.checklistbank.org/dataset/53548/taxon/411726)
[Usnea pallida Motyka, Usnea 2(1): 435 (1937)](https://www.dev.checklistbank.org/dataset/53548/taxon/412183)

**Comment by yroskov** [comment]
I only can say that CoL is expecting a new update for Species Fungorum from Paul Kirk in this month (March, keep fingers crossed). If Paul recognizes these names as valid species, they will appear in the global CoL in next 1-2 months after we receive update. 

Dear @collema, you may contact Paul directly and discuss validity of the species you are interested in.

**Comment by collema** [comment]
@yroskov Thank you for the reply and information!

**Comment by collema** [comment]
Actually, I think I expressed myself unclearly. It is not about the validity of the species. What I meant is that there are many taxon names that are included in Species Fungorum, but are missing (="taxon match none") from GBIF (and thus from COL?). I wonder why they are missing and if those can be added to COL? I do not understand your discussion above. I do not know what "bare names" are.

**Comment by yroskov** [comment]
"Bare name" is a term used in the Checklistbank. This status is applied to the names which are not recognized as valid taxa by taxonomists (or, at least by the editors of Species Fungorum). These names are present in the Index Fungorum (= nomenclator, a flat list of ever published scientific names), but absent in the Species Fungorum (= taxonomic checklist of species recognized by taxonomists as valid taxa with synonyms). Bare names must not be included in CoL. They are present only in the nomenclator.

**Comment by collema** [comment]
from above: "but absent in the Species Fungorum"
For example the species Anema jenisejense is now present in Species Fungorum: https://www.speciesfungorum.org/names/NamesRecord.asp?RecordID=546190

But still absent from GBIF. I wonder if there is another problem?
